Dockerfile

```diff
@@ -5,30 +5,26 @@
 # image with tools only needed during the image build.
 
 # First build the temporary image.
-FROM alpine:3.7 AS builder
+FROM python:3.7-slim-buster AS builder
 
 # Execute subsequent RUN statements with bash for handy modern shell features.
-RUN apk add --no-cache bash
 SHELL ["/bin/bash", "-c"]
 
 # Add system deps for building
-RUN apk add --no-cache \
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
```

Comment:
Common practice is to bundle the `apt-get update && apt-get install …` to reduce the size and number of layers.